topics/contributing/tutorials/updating_tutorial/tutorial.md

```diff
@@ -0,0 +1,166 @@
+
+time_estimation: 1H
+key_points:
+- The GTN has a number of fantastic features that make it a cutting-edge resource.
+- Jumping into existing materials can be daunting, but by following these steps, you can be a contribute to this vibrant community!
+
+contributions:
+  authorship:
+    - nomadscientist
+  editing:
+    - wee-snufkin
+
+requirements:
+-
+  type: "internal"
+  topic_name: contributing
+  tutorials:
+      - github-interface-contribution
+
+---
+
+Here, we provide a clear set of instructions for updating a GTN tutorial that uses the Galaxy interface for performing analysis. The GTN, and Galaxy, have a number of features to make it as reproducible and shareable as possible, so navigating what needs to be done for an overall update may feel daunting - but not anymore!
+
+We encourage you to pick a tutorial to try this on, so you can use this tutorial side-by-side.
+
+# Phase 1: Find a tutorial with an outdated workflow
+
+{% include _includes/cyoa-choices.html option1="No" option2="Yes" default="No"
+       text="Do you already have a tutorial and workflow you want to update?" %}
+
+<div class="No" markdown="1">
+
+> <hands-on-title>Check for an outdated workflow</hands-on-title>
+>
+> 1. **Find a {% icon hands_on %} Hands-on tutorial** in your training topic of interest (for example, Single-cell)
+> 2. **Select the {% icon workflow %} workflow** from the header
+> 3. **Import the workflow** to your server
+> 4. Go to the **Workflow** menu and select {% icon galaxy-wf-edit %} **Edit workflow**
+> 5. **Click through the tools** in the workflow and check the {% icon tool-versions %} to see if the tools are outdated.
+{: .hands_on}
+
+If the workflow has tools that are up to date (or very close!), great! That tutorial does not need updating! Try another one!
+
+</div>
+
+<div class="Yes" markdown="1">
+
+> <hands-on-title>Import the workflow</hands-on-title>
+>
+> 1. **Navigate to your target {% icon hands_on %} Hands-on tutorial**
+> 2. **Select the {% icon workflow %} workflow** from the header
+> 3. **Import the workflow** to your server
+> 4. Go to the **Workflow** menu and select {% icon galaxy-wf-edit %} **Edit workflow**
+{: .hands_on}
+
+</div>
+
+# Phase 2: Check that nobody else is working on this
+
+It's always a good idea to check, just in case!
+
+You can:
+ - Send a message to the [GTN Matrix Channel](https://matrix.to/#/#Galaxy-Training-Network_Lobby:gitter.im) is your quickest way forward.
+ - Search through the [GTN Github Repository](https://github.com/galaxyproject/training-material) for existing draft Pull Requests.
+ - Check with [individual communities](https://galaxyproject.org/community/sig/), who may have their own method of tracking. For example, the [🖖🏾Single-cell & sPatial Omics Community](https://galaxyproject.org/community/sig/singlecell/) have a shared [Click-Up board](https://sharing.clickup.com/9015477668/b/h/5-90152810734-2/557452707486fef) at the time of writing.
+
+# Phase 3: Update the workflow
+
```

Comment:
Could you add a sentence explaining that updating the workflow means using the last tool version? Thanks 😄